src/test/java/com/gachigang/ontherun/service/BusinessServiceTest.java

```diff
@@ -33,12 +35,20 @@ class BusinessServiceTest {
 
     @Test
     void testGetAllBusiness() {
-        businessService.getAllBusiness();
-        verify(businessRepository, times(1)).findAll();
+        int page = 0;
+        int size = 10;
+
+        PageRequest pageRequest = PageRequest.of(page, size);
+        when(businessRepository.findAll(pageRequest)).thenReturn(new PageImpl<>(Collections.emptyList()));
```

Comment:
We need to return some data in method mock and than assert that this data is present

Reply:
I`ve rewritten this test.